Profile Card UI Updated

blyncnov opened the pull request:
# Pull Request Preview 

* Mobile View

![localhost_3000_profile(iPhone X)](https://user-images.githubusercontent.com/66668114/194638843-6bea7ffa-4eec-4870-950e-5bf5c91d6646.png)

* Large Screen

![Screenshot 2022-10-07 at 20 16 40 (2)](https://user-images.githubusercontent.com/66668114/194638902-842e220a-69ef-422f-b02b-1d20b33bb169.png)
Screen

d-e-v-esh commented:
Looks really good 🔥🔥 
BTW you should remove the `yarn.lock` from staged changes before making a commit, otherwise it will overwrite the master `yarn.lock` file. The commit should contain only the files where you changed the code.

The quick solution will be that you can `undo` this commit, remove the `yarn.lock` file from the staged changes. then make back the commit and then do `git push -f`.

d-e-v-esh commented:
Also I would suggest that you make multiple smaller commits as you are adding or removing things as opposed to one large commit. It will make it easier to look through and track back if needed in future.

blyncnov commented:
Alright , will take note of those .. 

Literally you mean, if I make changes in profile.js , then only profile.js should be in my commit ?

blyncnov commented:
You can Review again

d-e-v-esh commented:
> Literally you mean, if I make changes in profile.js , then only profile.js should be in my commit ?

Yes! Exactly.

d-e-v-esh commented:
The `yarn.lock` is still in this Pull Request. This number at the top right is inflated becasue of the changes in the `yarn.lock`. I think you will have to ***undo commit***.
![image](https://user-images.githubusercontent.com/59534570/194692502-16614708-5dca-4d70-b1ec-db9f0e011ef4.png)

You will find lots of tutorials on YouTube on this topic that will help.

padmajabhol commented on src/components/Authentication/User-Profile/useprofile.js:
```diff
@@ -38,15 +51,58 @@ function Profile() {
+              <div className='profile-image-circle'></div>
+            </ProfileImageContainer>
+            <ProfileCTXText>
+              <div className='profile-ctx-user'>
```
Use styled components here instead of using h1/div tags.

padmajabhol reviewed (changes requested):
Please use styled components and convert the div and h1s. Also please delete the package -lock.json. We don't need to mix both npm and yarn packages. The UI looks good. Thanks a lot.

blyncnov commented:
I will do that

blyncnov commented:
Changes Made... I will wait back for your review.

Thanks

d-e-v-esh commented:
@blyncnov You'll have to `undo` the commits, yarn lock is still in the Pull Request.
![image](https://user-images.githubusercontent.com/59534570/194735566-a76f4d4c-bb7a-4ba0-b210-092704a4e53e.png)

blyncnov commented:
I thought I deleted that already.

<img width="1347" alt="Screenshot 2022-10-09 at 07 41 53" src="https://user-images.githubusercontent.com/66668114/194742356-0a652bb3-1976-4eb2-b020-14fc90d0b8a0.png">

blyncnov commented:
@d-e-v-esh maybe you can show me how else if you are free ?? 

Initially I uncommitted and push the file, I can't find the yarn.lock in my branch of repo tho and in the PR, i see deleted

d-e-v-esh commented:
https://user-images.githubusercontent.com/59534570/194747004-68657b87-2e32-436a-8902-6970681af3cb.mp4

@blyncnov I made a small video where I removed an image from a commit and re commited and pushed.

blyncnov commented:
![20221009_095444.jpg](https://user-images.githubusercontent.com/66668114/194747546-cef47831-0963-4c9c-a2f7-d089c3120341.jpg)

Great.. did that

blyncnov committed:
test removal

blyncnov commented:
Completed 👋

d-e-v-esh commented:
@blyncnov Great! Can you please rename the commit from `test removal` to something more descriptive?

blyncnov commented:
@d-e-v-esh Okay . Will do

blyncnov committed:
removed yarn.lock

blyncnov commented:
Did that work @d-e-v-esh  ??

d-e-v-esh commented:
No, descriptive meaning something that tells more about that you added or removed from the codebase

blyncnov commented:
@d-e-v-esh  

I should do that via the commit message? 

e.g

git commit -m "I improved user profile UI Card design bla bla bla   "

Right ??

d-e-v-esh commented:
@blyncnov Commit message has two parts, `title` and `description`. Look them up.

blyncnov committed:
Profile Card UI Updated

I Updated the user profile card UI Design,  and made edits, Styled components based styling

blyncnov commented:
@d-e-v-esh please check 

It's my first time with this really, and I am learning alot from you ..

Thanks

d-e-v-esh commented:
@blyncnov You can yourself check under the commits section.
![image](https://user-images.githubusercontent.com/59534570/194751016-49be7ec0-3166-4e26-8e99-fa01221b5d81.png)

You see there are 3 commits, we want to have all the changes under one commit which is the latest one you made. I think you should undo all commits and re-commit it with the new commit message. And remove yarn lock from it. You don't have to wait for me to check. You can yourself check under the commits tab in the Pull Request.
